src/pages/contest/create-contest.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from "react";
+import { useCreateContestMutation } from "<@>/store/contest/contest-api";
+
+const ContestForm = () => {
+  const [title, setTitle] = useState("");
```

Comment:
suggestion: to reduce the number of use states, what if you use a contest object and set its attributes like: 

const [contest, setContest] = useState<Contest>({});
setContest({ ...contest, title: "title" });